modules/api/src/main/java/org/eclipse/xpanse/api/config/ServiceTemplateEntityConverter.java

```diff
@@ -82,6 +82,8 @@ public static UserOrderableServiceVo convertToUserOrderableServiceVo(
             BeanUtils.copyProperties(serviceTemplateEntity, userOrderableServiceVo);
             userOrderableServiceVo.setServiceTemplateId(serviceTemplateEntity.getId());
             userOrderableServiceVo.setIcon(serviceTemplateEntity.getOcl().getIcon());
+            userOrderableServiceVo.setNamespace(
+                    serviceTemplateEntity.getOcl().getNamespace());
```

Comment:
This value should be serviceTemplateEntity.getNamespace() not the namespace in Ocl

Reply:
tested and removed.